### ISSUE TITLE
[BUG] In the bright theme, some text colors are not displayed clearly

### ISSUE THREAD

**Issue by SZooo** [neffo/bing-wallpaper-gnome-extension#191]
**Describe the bug**
As mentioned above. 

**Screenshots**

**To Reproduce**
1.Install a bright theme or adjust to bright mode.

**Desktop (please complete the following information):**
 - OS:  Ubuntu 22.04
 - Extension Version: 43
 - GNOME Version: 42.5

**Additional context**
In fact, I've found many extensions in Ubuntu that don't fit the bright mode display.

**Comment by neffo** [comment]
Can you share a screenshot?

Do you mean light style or a shell theme? (Does toggling this fix anything?)

![image](https://user-images.githubusercontent.com/5569112/204757954-0e5b2086-44d4-4dbb-90a6-e3d819287068.png)

**Comment by SZooo** [comment]
The text color is too light and blends into the background. Switch to a dark theme and text displays normally.

![图片](https://user-images.githubusercontent.com/8414397/204807512-a0d5d6c4-c25b-49bc-9f15-51dbcf186355.png)

**Comment by neffo** [comment]
Yeah, so just the shaded items. We may be able to fix that for this extension, but it might be an issue with that theme. Noting your icons (I don't recognize that icon theme) and the top task bar being white. What distro are you running?

**Comment by SZooo** [comment]
> Yeah, so just the shaded items. We may be able to fix that for this extension, but it might be an issue with that theme. Noting your icons (I don't recognize that icon theme) and the top task bar being white. What distro are you running?

I installed some extensions, so icon is a little different. I'm running ubuntu 22.04

**Comment by neffo** [comment]
Do you have GNOME tweaks installed?
```
sudo apt install gnome-tweaks
```
If so, what do you see here in themes?
![image](https://user-images.githubusercontent.com/5569112/205050550-19af77cf-d8a8-463e-a2df-192ca061dc77.png)

**Comment by SZooo** [comment]
it's pretty theme


![截图 2022-12-02 10-49-50](https://user-images.githubusercontent.com/8414397/205204213-0cddf279-0486-45aa-afb1-543e6b8fccca.png)

**Comment by neffo** [comment]
Yeah looking into this a bit, I've not tested that same theme but some Shell themes are incomplete (or broken) and result in missing or strangely coloured menus. One shell theme I tested resulted in switches being invisible. I don't recommend using them, or try updating them to the latest versions.
